homeworks/artur.kobyliatsky_Allocene/GamesProject/games.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+        <script src="https://kit.fontawesome.com/b3adce1055.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="./style/games.css">
+        <link rel="icon" type="image/x-icon" href="./img/logo.png">
+        <title>Games</title>
+    </head>
+
+    <body>
+        <header class="header">
+            <div class="header__box">
+                <div class="header__box-content">
+                    <div class="header__logo-section">
+                        <img src="./img/logo.png" alt="logo">
+                    </div>
+                    <div class="header__menu-section">
+                        <div class="header__menu-section">
+                            <ul class="menu__list">
+                                <li>
+                                    <a href="./games.html" class="menu__list-link">Games</a>
+                                </li>
+                                <li>
+                                    <a href="./about.html" class="menu__list-link">About</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="contain">
+                <div class="content">
+                    <div class="text">
+                        <h1>
+                            Study project "Games"
+                        </h1>
+                    </div>
+                    <div class="games__card">
+
+                        <div class="games__card--cod2">
+                            <div class="cod2__general">
+                                <div class="cod2__general--img">
+                                    <img src="./img/Call_of_Duty-_Modern_Warfare_II.jpeg" alt="COD MW2">
+                                </div>
+                                <div class="cod2__general--desc">
+                                    <h3>COD: MWII</h3>
+                                    <p>
+                                        Big changes come to the COD: MWII formula in this sequel...
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="cod2__description">
```

Comment:
It is also a list, isn't it? and please add alignment and othe rstyles

Reply:
Okay, I understand why they need UL instead of DIV, I changed everything, it’s ready